stack.yaml

```diff
@@ -1,6 +1,9 @@
 
-resolver: lts-18.14
-compiler: ghc-8.10.7
+resolver: nightly-2022-04-02
```

Comment:
Thanks @waivio.

Is changing the compiler to 9.2.2 necessary? That is the compiler of `nightly-2022-04-02`. And perhaps more importantly, is changing the snapshot necessary?

LH is using  `lts-18.27` and it has built `hashable-1.3.5.0` fine in my tests.